modules/fedimint-ln-client/src/receive.rs

```diff
@@ -38,6 +38,7 @@ const RETRY_DELAY: Duration = Duration::from_secs(1);
 /// ```
 #[derive(Debug, Clone, Eq, PartialEq, Hash, Decodable, Encodable)]
 pub enum LightningReceiveStates {
+    SubmittedOfferV0(LightningReceiveSubmittedOfferV0),
```

Comment:
You cannot prepend anything to this enum without breaking decoding for all following variants due to their index changing.

You could add the *new* `SubmitOffer` variant to the end, but I've described a better way in the next comment.